app/components/views/Home/index.tsx

```diff
@@ -65,6 +69,19 @@ const useWeb3Modal = () => {
             darkMode: false, // Optional. Use dark theme, defaults to false
           },
         },
+        torus: {
+          package: Torus,
+          options: {
+            networkParams: {
+              host: "matic", // optional
+            },
+          },
+          display: {
+            name: "Email, Google, Twitter",
+            description: "Use your existing services to connect.",
```

Comment:
is this too many chars?
```suggestion
            description: "Torus provides easy one-click social or email wallets",
```